app/templates/message_template.hbs

```diff
@@ -22,7 +22,7 @@
               {{display-message-time message.createdAt}}
             </div>
 
-            {{message.body}}
+            {{{message.body}}}
```

Comment:
ok so this means the string is not safe... opens up potential XSS attacks. I'll open a new ticket to find any problem areas.